src/libraries/UQ160x96.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.19;
+
+// a library for handling binary fixed point numbers (https://en.wikipedia.org/wiki/Q_(number_format))
+
+// range: [0, 2**160 - 1]
+// resolution: 1 / 2**96
+
+library UQ160x96 {
+    //solhint-disable-next-line state-visibility
+    uint224 constant Q96 = 2 ** 96;
```

Comment:
`uint224` and `uint256` are both used here. For consistency, would it be worth choosing one?

Reply:
yes I think uint256 makes more sense for sake of consistency, although either doesn't make a difference, updated in a7a23ba